contrib/win32/win32compat/misc.c

```diff
@@ -68,7 +68,9 @@ static char* s_programdir = NULL;
 
  /* Difference in us between UNIX Epoch and Win32 Epoch */
 #define EPOCH_DELTA_US  116444736000000000ULL
```

Comment:
Looks like this is not in us (micro seconds), rather in 100ns units. Feel free to fix the name or have this instead:
#define EPOCH_DIFF_SECONDS 11644473600ULL

Reply:
Yes, you're right. I suggest to rename it in "EPOCH_DELTA". The constant also using in "unix_time_to_file_time" and "file_time_to_unix_time" functions.